oracle/types/candle_price.go

```diff
@@ -14,15 +14,15 @@ type CandlePrice struct {
 }
 
 // NewCandlePrice parses the lastPrice and volume to a decimal and returns a CandlePrice
-func NewCandlePrice(provider, symbol, lastPrice, volume string, timeStamp int64) (CandlePrice, error) {
+func NewCandlePrice(lastPrice, volume string, timeStamp int64) (CandlePrice, error) {
```

Comment:
Is there a reason to not pass in the provider anymore for a more descriptive log?